iotagent-manager/latest/docker-compose.yml

```diff
@@ -0,0 +1,27 @@
+version: '3'
+
+services:
+  iotmanager:
+    image: iotmanager
+    ports:
+      - ${IOTA_SERVER_PORT}:${IOTA_SERVER_PORT}
```

Comment:
All these ${X} of the compose should have a default that works with the default deployment we've been using in mastermind and smartSDK. For example, ${IOTA_SERVER_PORT:-8082} and so on.

This is for those that might use the compose without mastermind :)

Reply:
@taliaga I don't recall if the Service Manager was fixed to account for those, because it used to fail while parsing the compose when it encountered that syntax. I'll fix the recipes once that is taken care of properly.